src/databrowser.js

```diff
@@ -132,13 +131,14 @@ define([
                 state: {opened: !node.recursive && children.length <= MAX_OPEN_NODE},
                 children: children,
                 data: {
-                    handleDrop: _.partial(handleDrop, path, node.sourceInfo),
+                    handleDrop: _.partial(handleDrop, node, node.sourceInfo),
                     getNodes: node.recursive ? getNodes : null,
                 }
             };
         }
-        function handleDrop(path, info, target) {
-            var widget = widgets.util.getWidget(target, vellum);
+        function handleDrop(node, info, target) {
+            var widget = widgets.util.getWidget(target, vellum),
+                path = widget.mug.form.richText && node.hashtag || node.xpath;
```

Comment:
It's possible I wrote this code and totally forgot but what's the advantage of `... && ... || ...` over ` ... ? ... : ...` ?